### ISSUE TITLE
Build fails with clang-cl due to undefined symbols

### ISSUE THREAD

**Issue by emmenlau** [google/glog#575]
I've built glog 0.4.0 often with cmake on various platforms successfully. We just recently switched to clang-cl on MSVC and now the build fails for the first time, with several errors about undefined symbols such as:
```
LINK Pass 1: command "C:\PROGRA~2\MICROS~1\2019\COMMUN~1\VC\Tools\Llvm\x64\bin\lld-link.exe CMakeFiles\demangle_unittest.dir\src\demangle_unittest.cc.obj /out:demangle_unittest.exe /implib:demangle_unittest.lib /pdb:demangle_unittest.pdb /version:0.0 /machine:x64 /MACHINE:X64 /DEBUG /debug /INCREMENTAL /subsystem:console glogd.lib dbghelp.lib C:\data\stable-artifacts-MSVC-Haswell-7-x64-cl19.27.29111_clang10.0.0\Debug\lib\gflags_debug.lib shlwapi.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\demangle_unittest.dir/intermediate.manifest CMakeFiles\demangle_unittest.dir/manifest.res" failed (exit code 1) with the following output:
 lld-link: error: undefined symbol: bool __cdecl google::Demangle(char const *, char *, int)
```

Does that mean anything to anyone?

**Comment by sergiud** [comment]
Please test the version from master.

**Comment by emmenlau** [comment]
Sadly, that's not easy for me due to the way our build works in a continuous integration.

**Comment by sergiud** [comment]
May be fixed by #619.
